src/events/message-reaction-add/message-report.ts

```diff
@@ -0,0 +1,65 @@
+import { EmbedBuilder } from "discord.js";
+import type { MessageReactionAddEvent } from "../../handlers/messageReactionAddEvents.handler";
+
+const reportEmojis = process.env.REPORT_EMOJIS.split(",");
+const reportEmojiAmount = Number(process.env.REPORT_EMOJI_AMOUNT);
+const reportChannelId = process.env.REPORT_CHANNEL_ID;
+
+const event: MessageReactionAddEvent = {
+  execute: async (client, reaction) => {
+    if (!reportEmojis.length || !reportEmojiAmount || !reportChannelId) return;
```

Comment:
Can you add `console.error` at here as well? Then the operation person will note that they miss the env var.

Reply:
done